### PR TITLE
Add search to Get Stream Views

### PR DESCRIPTION
**Pull request by TomSeymour**

### CODE REVIEW

File: Documentation/SequentialDataStore/SDS_Views.md [hunk 1]
```diff
@@ -613,27 +613,46 @@ Returns the stream view map corresponding to the specified streamViewId within a
 
 Returns a list of stream views within a given namespace.
 
+If specifying the optional query parameter or optional filter parameter, the list of stream views returned is filtered to match 
+the search/filter criteria. If neither parameter is specified, the list includes all stream views 
+in the Namespace. See [Searching](xref:sdsSearching) 
+and [Filter Expressions: Metadata Objects](xref:sdsFilterExpressionsMetadata)  
+for information about specifying those respective parameters.
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/SDS_Views.md]: This spacing on this paragraph turns out a bit strange.

Comment (TomSeymour) [reply Documentation/SequentialDataStore/SDS_Views.md]: Yes, I've seen that.  It actually does the same thing in Sds_Streams but not in Sds_Types.  I've tried manipulating it but it comes back the same way, any suggestions are welcome!

Comment (osimarkrogers) [reply Documentation/SequentialDataStore/SDS_Views.md]: Two spaces is supposed to create a line break in markdown - - try deleting the break between the end of the link (xref:sdsFilterExpressionsMetadata) and "for information," and then entering exactly one space between them.  That may restore normal wrapping.

Comment (osimarkrogers) [reply Documentation/SequentialDataStore/SDS_Views.md]: I just tried this in master/preview and it worked - - apparently there were two spaces preceding "for information," which signals a line break.  Just take them out and replace with only one, and it should fix it.

File: Documentation/SequentialDataStore/SDS_Views.md [hunk 2]
```diff
@@ -644,7 +663,43 @@ Returns a list of stream views within a given namespace.
...
+  Sample response body:
+
+```json
+      HTTP/1.1 200
+      Content-Type: application/json
 
+       [  
+         {  
+            "Id":"StreamView",
+            "Name":"StreamView"
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/SDS_Views.md]: Now you get hints!

Missing a comma:  
`"Name":"StreamView",`

File: Documentation/SequentialDataStore/SDS_Views.md [hunk 3]
```diff
@@ -644,7 +663,43 @@ Returns a list of stream views within a given namespace.
...
+               {  
+                  "SourceId":"Time",
+                  "TargetId":"Time"
+               },
+               {
+                  "SourceId":"Status",
+                  "TargetId":"Status"
+               },
+               {
+                  "SourceId":"Measurement"
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/SDS_Views.md]: comma missing

File: Documentation/SequentialDataStore/SDS_Views.md [hunk 4]
```diff
@@ -644,7 +663,43 @@ Returns a list of stream views within a given namespace.
...
+            "Id":"StreamView",
+            "Name":"StreamView"
+            "SourceTypeId":"Simple",
+            "TargetTypeId":"Simple3"
+         },
+         {  
+            "Id":"StreamViewWithProperties",
+            "Name":"StreamViewWithProperties",
+            "SourceTypeId":"Simple",
+            "TargetTypeId":"Simple3"
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/SDS_Views.md]: comma missing

File: Documentation/SequentialDataStore/SDS_Views.md (hunk 1 above)
Comment (mckenzielee) [reply Documentation/SequentialDataStore/SDS_Views.md]: I would just remove all line breaks. Let the browser worry about spacing.

Comment (TomSeymour) [reply Documentation/SequentialDataStore/SDS_Views.md]: Thanks Mark, removed spaces.

**Pull request merged**